tests/test_backends_qibotn.py

```diff
@@ -1,16 +1,18 @@
 import os
 
 import qibo
+from qibo.backends import _Global
 
 # Force quimb to use qibojit default number of threads.
+_Global.backend()
```

Comment:
Pretty useless statement?

Reply:
Since the next line

```py
os.environ["NUMBA_NUM_THREADS"] = f"{qibo.get_threads()}"
```

requires the initialization of `_Global._backend`, the previous `GlobalBackend` had a `__new__()` method that did the initialization.

Reply:
Yes, then don't use `_Global.get_backend()` in `get_threads()`, but rather the `_Global.backend()`.

This should solve both problems. Here, and anywhere the `get_threads()` function will be called :D

Reply:
Updated it. In `get_threads`, `qibo.get_backend()` is used and it calls `backend`.